tasks/otel_agent.py

```diff
@@ -40,7 +40,7 @@ def byoc_release(ctx, image=DDOT_DEV_AGENT_TAG, branch=DDOT_DEV_AGENT_BRANCH, re
 
 
 @task
-def build(ctx):
+def build(ctx, byoc="false"):
```

Comment:
IMHO opinion this should be a boolean
```suggestion
def build(ctx, byoc="false"):
```

Reply:
I can't see a change in the suggestion, but if you are suggesting:
```
def build(ctx, byoc=False):
```

I don't think invoke can convert to bool:

```
➜  datadog-agent git:(mackjmr/byoc-redapl) ✗ inv otel-agent.build --byoc=false && ./bin/otel-agent/otel-agent run --config collector.yaml --core-config acfg.yaml
No idea what 'false' is!

➜  datadog-agent git:(mackjmr/byoc-redapl) ✗ inv otel-agent.build --byoc=False && ./bin/otel-agent/otel-agent run --config collector.yaml --core-config acfg.yaml
No idea what 'False' is!
```

Reply:
I meant exactly that:
```
def build(ctx, byoc=False):
```

I think invoke does support it, but it becomes something like `--byoc` or `--no-byoc`, or something like that. I'm almost sure boolean flags are supported.